src/storage/engine/mod.rs

```diff
@@ -605,46 +597,65 @@ mod tests {
         })
     }
 
+    #[derive(PartialEq, Eq, Clone, Copy)]
+    enum SeekMode {
+        Normal,
+        Reverse,
```

Comment:
`near_reverse_seek` is still using. I prefer keeping `reverse_seek` in case it's needed again in the future.